src/App.tsx

```diff
@@ -1,31 +1,204 @@
-import React from "react";
+import React, { Component } from "react";
 import logo from "./logo.svg";
 import "./App.css";
 import { createSecureContext } from "tls";
 import { create } from "domain";
 import { bindKeyboardListeners } from "./instruments";
+import { Stage, Layer, Ellipse } from "react-konva";
+import CSSTransitionGroup from "react-transition-group";
 
 bindKeyboardListeners();
 
-const App = () => {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const sweepLength = 1;
```

Comment:
Oo. You should delete all of this synth code and just import it from the other file

Reply:
The new function you want to import and call instead of audio is `bindEventListeners`